package.json

```diff
@@ -23,18 +23,20 @@
     "doctor": "npm run check-node-version && ts-node bin/doctor",
     "check-node-version": "check-node-version --node \">= 12\"",
     "lint": "npm-run-all check-node-version lint:es lint:ts lint:sh lint:md",
-    "lint:es": "eslint --ignore-pattern node_modules/ --ignore-pattern fixtures/ \"{bin,examples,src,scripts,tests}/**/*.ts\"",
+    "test:lint": "npm-run-all check-node-version test:lint:es lint:ts lint:sh lint:md",
+    "test:lint:es": "eslint -c tests/.eslintrc.js --fix --ignore-pattern node_modules/ --ignore-pattern fixtures/ \"**/*.spec.ts\"",
+    "lint:es": "eslint --fix --ignore-pattern node_modules/ --ignore-pattern fixtures/ --ignore-pattern \"**/*.spec.ts\"  \"{bin,examples,src,scripts}/**/*.ts\"",
```

Comment:
Please do not use '--fix' implicitly.

If you feel need to add a fix script, I will be good to add a 'lint:fix'